docs/developer-docs/release-procedure.md

```diff
@@ -0,0 +1,77 @@
+# Release Procedure
+
+This document is a Core team guide, provided publicly to ensure transparancy in the release process.
+
+## Versioning
+
+Jellyfin uses a 3-digit semantic versioning scheme. All releases will have versions in the `X.Y.Z` format, starting from `10.0.0`. Note however that the `10.X.Y` release chain represents the "cleanup" of the codebase, so it should be accepted that `10.X.Y` breaks all compatibility, at some point, with previous Emby-compatible interfaces.
+
+#### `X` - Major breaking versions
+
+The first digit represents a consistent release without breaking 3rd party interfaces including APIs.
+
+This value will be increased if a release:
+
+* Breaks compatibility with the 3rd party API.
+* Breaks compatibility with the plugins API.
+* Makes significant enough changes to warrant a major version bump, as decided by the Core team.
+
+#### `Y` - Minor versions
+
+The second digit represents feature releases that do not warrant major version increases.
+
+This value will be increased if a release:
+
+* Introduces new features.
+* Makes minor non-breaking API changes.
+
+#### `Z` - Hotfix versions
+
+The third digit represents hotfix releases that do not include additional features.
+
+This value will be increased if a release:
+
+* Introduces bugfixes or otherwise changes `master` branch code since the last release.
+
+## General Release Philosophy
+
+We will be following the guide [A successful Git branching model](https://nvie.com/posts/a-successful-git-branching-model/) for our release branching model and workflow, with GitHub-specific tweaks as required.
+
+#### Significant deviations
+
+* The `develop` branch is named `dev`.
+
+* Developers should, as described in [the contributing guidelines](/developer-docs/contributing-guidelines), do feature branch work on their own forked copies of repositories.
+
+* Standard review requirements as defined for each repo always apply.
+
+## Release procedure
+
+1. Announce release in the [jellyfin-dev](https://matrix.to/#/#jellyfin-dev:matrix.org) Matrix/Riot channel at least 24h before creating the release branch.
+
+1. Ensure any required PRs are merged into both the [jellyfin](https://github.com/jellyfin/jellyfin) and [jellyfin-web](https://github.com/jellyfin/jellyfin-web) repositories.
+
+1. Create a release branch for the [jellyfin](https://github.com/jellyfin/jellyfin) and [jellyfin-web](https://github.com/jellyfin/jellyfin-web) repositories with the format `release-X.Y.Z`, where `X.Y.Z` is the new version number, based off the current `dev` branches.
+
+1. Execute the `bump_version` script inside the release branch in the [jellyfin](https://github.com/jellyfin/jellyfin) local repository. Commit the resulting differences as `Bump version to X.Y.Z`.
```

Comment:
Does this update Jellyfin/Jellyfin to use the later commit from Web?

Reply:
Yes, the script (once merged) attempts to check out the same branch as `master`, the idea being it will be on `release-X.Y.Z` on the main repo, and so check out the `release-X.Y.Z` branch in `jellyfin-web` as well. Both branches are created and will merge together.